html: removed left rails

thelostone-mc opened the pull request:
<!--
  Thank you for your pull request. Please provide a description above and review
  the requirements below.

  Contributors guide: https://github.com/gitcoinco/web/blob/contributing/CONTRIBUTING.md
-->

##### Description
<!-- A description on what this PR aims to solve -->
- removed left panel in /profile, /toolbox, /new/funding
- replaces newletter and pushed it to the bottom

##### Checklist
<!-- Remove items that do not apply. For completed items, change [ ] to [x]. -->

- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
<!-- Provide affected core subsystem(s) (like doc, ui, crypto, etc). -->



##### Refers/Fixes
Fixes: https://github.com/gitcoinco/web/issues/411
<!--
  Link to an issue if applicable. For example:
  If your PR fixes an issue  -> Fixes: #102
  If your PR refers an issue -> Refs: #101
-->

thelostone-mc commented:
@PixelantDesign  https://github.com/gitcoinco/web/issues/411#issuecomment-368020251

thelostone-mc commented:
## Screenshots


### /profile

![screenshot-2018-2-25 thelostone-mc gitcoin](https://user-images.githubusercontent.com/5358146/36639691-6bcd8096-1a38-11e8-896f-1f2493b1b4ea.png)

### /funding/new

![screenshot-2018-2-25 gitcoin push open source repos forward](https://user-images.githubusercontent.com/5358146/36639692-6c0b0fa6-1a38-11e8-9bc3-513e51af6885.png)

### /toolbox

![screenshot-2018-2-25 gitcoin toolbox](https://user-images.githubusercontent.com/5358146/36639693-6c5c5456-1a38-11e8-9942-f944a053e38f.png)

thelostone-mc commented:
@owocki / @mbeacom  :  my submission didn't go through !!

Bounty: https://gitcoin.co/funding/details?url=https://github.com/gitcoinco/web/issues/411
Initially it gwei was set to be 0.1 gwei and that took too long

Initial submission (where the gwei was set to 0.1gwei by default ) : https://etherscan.io/tx/0x1762e7025519611b59924a6abf56c2acaf45cd299f346661c28620ee6d306cb2

Successful submission (where I raised it to 0.4 gwei ) : https://etherscan.io/tx/0xf0c3a85e9cbcab3d8f5fe824cc795e546a7707ba426cadae05a0543acd35fb5b

owocki commented:
> It took close to 30min for my submission to turn up on the bounty.
> Worth creating an issue -> to see if it can be updated faster ?

unfortunately, i think this is just a consequence of working with the ethereum mainnet.  hopefully scaling / sharding makes things faster in the future!

owocki commented:
@PixelantDesign did you want this to go live on the site?  do we want to remove the sidebar on ALL interior pages to the site, or just the ones you specified?

@thelostone-mc how confident do you feel that you've handled any responsiveness issues (mobile, tablet, etc)?

thelostone-mc commented:
@owocki No issues rendering in mobile /tablet devices ! 
Just cause you asked  (I added a few extra lines off CSS to prettify the content on the mobile )
^That doesn't have anything to do with this PR but I added that anyways  -> Hope that's alright ^_^

owocki commented:
makes sense to me @thelostone-mc 

@PixelantDesign let me know if youd like these changes live on the site and if you feel like they should apply to more interior pages than just the three specified in the desc ( /profile, /toolbox, /new/funding )

PixelantDesign commented:
@owocki I think we can push it live.

thelostone-mc commented:
@owocki all good!!

owocki commented:
hey @thelostone-mc -- i was just chatting with @PixelantDesign on slack..  could you make sure that we've made this change to all interior pages ( tools, profile, leaderboard, submit funding, kill funding, accept funding … etc ) instead of just the ones we originally specified (  /profile, /toolbox, /new/funding )

sorry for the scope creep here.  it was a miscommunication internally.. hopefully should not be too much more work to do this while its still top of mind

thelostone-mc commented:
@owocki Made the changes for everything except `leadeboard` as I'm having trouble populating data (what to where)


## Screenshots

### /process

![screenshot-2018-2-27 gitcoin push open source repos forward](https://user-images.githubusercontent.com/5358146/36748521-161f3558-1c1e-11e8-8c24-2a2a1d983e26.png)

### /kill

![screenshot-2018-2-27 gitcoin push open source repos forward 1](https://user-images.githubusercontent.com/5358146/36748522-16659714-1c1e-11e8-8dfc-d0e749913a8f.png)


### /fulfill
![screenshot-2018-2-27 gitcoin push open source repos forward 2](https://user-images.githubusercontent.com/5358146/36748525-16a5fc3c-1c1e-11e8-9b68-858401fdd902.png)

thelostone-mc commented:
## Screenshots

### /leaderboard

![screenshot-2018-2-28 leaderboard top payers](https://user-images.githubusercontent.com/5358146/36802819-f1cb0344-1cdb-11e8-84d0-2e94dcda1b0a.jpg)


@PixelantDesign @owocki Done and dusted :D 

@owocki thanks for the data population # Need to read docs more carefully 😅

PixelantDesign commented:
Looks great! Thanks @thelostone-mc!

PixelantDesign commented:
@thelostone-mc I found one more screen! Can we please remove? Thanks!

<img width="1436" alt="screen shot 2018-03-01 at 10 33 20 am" src="https://user-images.githubusercontent.com/238235/36853365-469927ba-1d3c-11e8-81d7-99d771b3b26f.png">

mbeacom commented on app/dashboard/views.py:
```diff
@@ -575,6 +576,7 @@ def profile(request, handle):
     params = {
         'title': 'Profile',
         'active': 'profile_details',
+        'newsletter_headline': "Be the first to know about new funded issues.",
```
Just a nitpick note...  Please be mindful of maintaining consistency with usage of `'` and `"`

thelostone-mc replied:
Ah my bad !!

mbeacom replied:
No worries - Thanks for the contribution!

thelostone-mc commented:
# Screenshots

## /loading Page
<img width="1440" alt="screen shot 2018-03-02 at 11 38 46 am" src="https://user-images.githubusercontent.com/5358146/36885941-f721fd5e-1e0f-11e8-92c8-ed795667dfae.png">


@PixelantDesign @mbeacom changes done :)

darkdarkdragon commented:
Issue Status: 1. **Open** 2. Started 3. Submitted 4. Done 

<hr>

__This issue now has a funding of 0.001 ETH (0.69 USD @ $692.1/ETH) attached to it.__

 * If you would like to work on this issue you can 'start work' [on the Gitcoin Issue Details page](http://localhost:8000/issue/gitcoinco/web/486/455).
 * Questions? Checkout <a href='https://gitcoin.co/help'>Gitcoin Help</a> or the <a href='https://gitcoin.co/slack'>Gitcoin Slack</a>
 * $0 more funded OSS Work available on the [Gitcoin Issue Explorer](https://gitcoin.co/explorer)
